vividus-extension-selenium/build.gradle

```diff
@@ -32,6 +33,11 @@ project.description = 'Vividus extension for selenium'
      implementation(group: 'org.apache.commons', name: 'commons-lang3', version: versions.commonsLang3)
      implementation(group: 'org.springframework', name: 'spring-core', version: versions.spring)
 
+     // browserup-proxy depends on both netty-all and netty-core, so we need to align versions of all netty dependencies
+     // https://github.com/browserup/browserup-proxy/blob/26714e26fae728733a13a0f332bd0c6dcaa78de5/browserup-proxy-core/build.gradle#L60-L76
+     implementation(group: 'io.netty', name: 'netty-all', version: "${nettyVersion}")
```

Comment:
Why don't we write version as for `seleniumVersion` or `browserupProxyVersion` ?